src/features/courses/course-controller.ts

```diff
@@ -2412,6 +2413,77 @@ class CourseController {
         });
     }
 
+    async canUserGetQuestions(options: CanUserGetQuestionsOptions): Promise<CanUserGetQuestionsResult> {
+        const {userId, courseTopicContentId, studentTopicAssessmentInfoId} = options;
+        let message = '';
+
+        let topic: CourseTopicContent | null = null;
+        let version: StudentTopicAssessmentInfo | null = null;
+
+        const user = await userController.getUserById(userId);
+
+        if (_.isNil(courseTopicContentId)) {
+            message = 'Sure, why not?';
```

Comment:
This isn't user-facing, right? 😂

Reply:
nope - I'm just not sure who's querying getQuestions without a topicId given... I'm not going to stand in their way ;P